ldm/modules/encoders/modules.py

```diff
@@ -144,9 +144,8 @@ def __init__(self, arch="ViT-H-14", version="laion2b_s32b_b79k", device="cuda",
                  freeze=True, layer="last"):
         super().__init__()
         assert layer in self.LAYERS
-        model, _, _ = open_clip.create_model_and_transforms(arch, device=torch.device('cpu'), pretrained=version)
```

Comment:
Is this just a cosmetic change (to remove unused variables), or this this needed for any functional changes?

I'd avoid too much divergence from the stability codebase, even if we don't really agree with the coding style. Ideally I would not have a custom fork at all.

Reply:
create_model_and_transforms loads and create image transforms for img2img but they don't use it the image transforms.
It's faster to just call create_model and only process the part that is needed.
That change is already in a PR waiting to be merged https://github.com/Stability-AI/stablediffusion/pull/52